android/app/src/main/res/layout/post_feedback_dialog.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout android:layout_height="wrap_content"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="wrap_content"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:orientation="vertical">
+    <TextView
+        android:id="@+id/post_feedback_header"
+        style="@style/CardContent.Header"
```

Comment:
@iamareebjamal this isn't the header style?

Reply:
But it isn't standard Material Dialog Header I think

Reply:
I am not sure, but this is used throughout the project (copyright, call for speakers) 
Do you want to change these to something like this?
<img width="279" alt="screen shot 2018-05-16 at 12 25 38 pm" src="https://user-images.githubusercontent.com/16715499/40101201-55b06296-5904-11e8-9eef-cae38f49a0fc.png">

Reply:
Yup. And well, the codebase is not exactly according to the best practices

Reply:
Will make title standard here and create an issue for changing others also 👍